DwmLutGUI/DwmLutGUI/Injector.cs

```diff
@@ -20,7 +20,7 @@ internal static class Injector
 
         static Injector()
         {
-            var basePath = Environment.ExpandEnvironmentVariables("%SYSTEMROOT%\\Temp\\");
+            var basePath = Path.GetTempPath();
```

Comment:
@Crevax Add `+ "\\"` at the end, otherwise it won't work

Reply:
Thanks, good catch! I was part-way through using `Path` methods instead of string concatenation, but after seeing a bunch more people in a Discord post issues I wanted to get this out quick.

Reply:
@Crevax Going to test it today and I'll approve the PR